lib/tasks/metrics.rake

```diff
@@ -0,0 +1,12 @@
+##
+# The namespace used by the tasks that handles the Github event metrics.
+#
+# Call the task from the command line with
+#
+#       rake metrics:process
+namespace :metrics do
+  desc 'Processes the stored events and generates the all the metrics'
+  task process: :environment do
+    ProcessMetricsJob.new.perform
```

Comment:
ActiveJob is meant to use perform_now or perform_later, not instantiating the obeject or anything. `ProcessMetricsJob.perform_now`